elements/bulbs-video/components/revealed.js

```diff
@@ -269,6 +245,14 @@ export default class Revealed extends React.Component {
     GoogleAnalytics.init(player, videoMeta.gaPrefix);
     Comscore.init(player, global.BULBS_ELEMENTS_COMSCORE_ID, videoMeta.player_options.comscore.metadata);
 
+    if (this.props.onPlay) {
```

Comment:
We should ask @spra85 about how he tested the jw events for the GoogleAnalytics plugin above.